wGetStarted/L-Dev-SimpleDataSave.md

```diff
@@ -510,46 +484,40 @@ This section will take the XML data sent by Interject from your report and put i
 
 ### Validations
 
-Now that you have all the data from your report into a table, you can do some validations on it before your target data source is updated. This section will run through certain validations and marks the \_MessageToUser parameter upon a check. This is example code. You can use and modify it to match your report and data or you can delete it. There are 4 validation checks in the example code:
+Now that you have all the data from your report into a table, you can do some validations on it before your target data source is updated. This section will run through certain validations and marks the \_MessageToUser upon a check. This is example code. You can use and modify it to match your report and data or you can delete it. There are 5 parts:
```

Comment:
maybe it's just looking weird in the code view here, but it looks like it says there are five parts and then only has four items in the list, and also every item on the list is numbered 1

Reply:
Yes it is looking weird in the code. There are 5 items, line 489-493.

Listing items with a 1. is a markdown thing, it will be converted to a numbered list.